test/util/testnode/config.go

```diff
@@ -128,8 +128,8 @@ func DefaultConfig() *Config {
 		WithAppOptions(DefaultAppOptions()).
 		WithAppCreator(cmd.NewAppServer).
 		WithSuppressLogs(true).
-		WithConsensusParams(DefaultConsensusParams()).
-		WithSuppressLogs(true)
+		// TODO: consider removing this line because default consensus params is invoked above.
+		WithConsensusParams(DefaultConsensusParams())
```

Comment:
I think we should remove this

Reply:
https://github.com/celestiaorg/celestia-app/issues/3684